### PR TITLE
Added syslog and  signal handling

### PR DESCRIPTION
**Pull request by colin-adams**
Sorry for the pull request on the master branch. I always forget to create a new branch after first forking.

### CODE REVIEW

File: posix.ipkg [hunk 1]
```diff
@@ -3,6 +3,14 @@ package posix
 sourcedir = src
 modules = System.Posix.Directory
         , System.Posix.Time
+	, System.Posix.Signal
+	, System.Posix.Syslog
+	, System.Posix.Test.Signal
+	, System.Posix.Test.Syslog
```
Comment (Melvar) [inline posix.ipkg]: It looks like you have hard tabs here. That might cause problems, but I don’t know how the ipkg parser reacts to them.

Comment (colin-adams) [inline posix.ipkg]: I encountered no problems, but I can change to spaces.

Comment (Melvar) [inline posix.ipkg]: Okay, well, in my browser it appears the tabs are displaying equal to seven spaces for some reason, so it caught my eye immediately. And in my mail client they appear equal to five spaces. I have no idea why.

Comment (colin-adams) [inline posix.ipkg]: I expect the parser must be whitespace oblivious. Anyway, I committed the change to only using spaces.

**Comment by colin-adams** [comment]
Can someone merge this PR now? I believe I have fully addressed all the comments.

**Comment by superfunc** [comment]
cc @puffnfresh

**Pull request merged**